From 76e548d09195de750a532ee568171b931c1216fe Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Thu, 18 Feb 2010 14:19:21 -0600
Subject: [PATCH 14/25] Don't enable ROOTLESS_WORKAROUND, it breaks composite

This possibly brings back whatever the bug is in
http://bugs.freedesktop.org/show_bug.cgi?id=1168 for
-rootless mode, but since we don't have reproduction
steps for that, I can't test that...

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
Tested-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
Signed-off-by: Yaakov Selkowitz <yselkowitz@users.sourceforge.net>
(cherry picked from commit 20701522be803fe47e921fcf059dadf64c7f287d)
---
 configure.ac |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git xorg-server-1.8.0/configure.ac xorg-server-1.8.0/configure.ac
index 3176790..f4a7b34 100644
--- xorg-server-1.8.0/configure.ac
+++ xorg-server-1.8.0/configure.ac
@@ -1860,7 +1860,6 @@ if test "x$XWIN" = xyes; then
 			dnl if we have windowswmproto, build rootless extension for multwindowextwm mode
 			if test "x$WINDOWSWM" = xyes ; then
 			       AC_DEFINE(ROOTLESS,1,[Build Rootless code])
-			       CFLAGS="$CFLAGS -DROOTLESS_WORKAROUND"
 			fi
 			;;
 		mingw*)
-- 
1.7.0.4

